gradle/dependency-management/agp-versions.properties

```diff
@@ -1,4 +1,4 @@
 # Generated - Update by running `./gradlew updateAgpVersions`
-latests=7.3.1,7.4.2,8.0.2,8.1.4,8.2.2,8.3.2,8.4.2,8.5.2,8.6.1,8.7.0,8.8.0-alpha04
-nightlyBuildId=12448304
-nightlyVersion=8.8.0-dev
+latests=7.3.1,7.4.2,8.0.2,8.10.0-alpha01,8.2.2,8.3.2,8.4.2,8.5.2,8.6.1,8.7.3,8.8.0,8.9.0-alpha09
```

Comment:
❓ Is it expected that this list is sorted lexicographically, not by version rank?

Reply:
Good catch. Fixed in 31755b234b8322ad6e7848ecff20e013241a70ab

Turns out the website from which we fetch versions got updated since my last time running the task so I also updated its logic to acommodate. And a new version just got published ...